api/inc/vmpu_exports.h

```diff
@@ -155,6 +155,27 @@ typedef struct
     uint32_t acl_count;
 } UVISOR_PACKED UvisorBoxConfig;
 
+typedef struct
+{
+    /* pointer to the user context */
+    void *ctx;
+    /* pointer to the box heap */
+    void *process_heap;
+    /* size of the box heap */
+    uint32_t process_heap_size;
+    /* pointer to the currently active heap */
+    void *active_heap;
+    /* cache of the box_id */
+    int box_id;
+
+    /* id of the mutex */
```

Comment:
How is the mutex used and why it must be here? Other elements in this struct are pretty self-explanatory but maybe this one needs some additional comment.